### DISCUSSION TITLE
A picture says more than a 1000 words...

### DISCUSSION THREAD

**Discussion by passas** [geany/geany#3746] [Ideas]
Hello, my name is Miguel, I am from Portugal.

I saw you'r repository on Trends. Opened up and saw no picture of the user interface of the IDE.

As a newbie, but enthusiast, I would like to be able to see the product before "buying" it. Can you please make a section on the README.md where you expose the product or make a folder with images from the product?

PS: The title is a proverb here in Portugal.

Best Regards,

Sérgio Costa

**Comment by passas** [comment]
\* "A picture is worth a thousand words"

**Comment by elextr** [comment]
See the project website at https://www.geany.org/

**Comment by elextr** [comment]
PS whats "Trends"?  And I wonder how Geany came to be trendy? :grinning:

**Reply by rdipardo** [reply]
Sounds like one of those aggregators that decorate listings with the repository's [social media preview image][0], which `geany/geany` doesn't have . . . yet?

<div align="center">
<img alt="gh-topics-geany" src="https://github.com/geany/geany/assets/59004801/ebd829a9-ce56-4408-b4a9-86f90752f735" width="400"/>
</div>

[0]: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview

**Reply by eht16** [reply]
Ha, https://github.com/topics/geany: we've got fancy new social media preview image. I don't understand how is that related to social media but I don't need to understand this I guess :).
The image is displayed on the search results now. THe dimensions are not ideal but good enough for now.

Better screenshots are welcome, according to the Github docs they should be in 1280×640px.

**Comment by eht16** [comment]
To the original issue:
I think this is a great idea, I find myself moaning on others' project pages when there is no screenshot easily accessable but failed to noticed it here :(.

What do you think of https://github.com/eht16/geany/tree/6e2dee3cf3df8d2d446d99c3241fc4bfc5cb8758? It doesn't look too bad to me.

Alternatively, we could move the screenshot below the "Features" section.

**Reply by passas** [reply]
Awesome 👌🏻

**Comment by elextr** [comment]
> I don't understand how is that related to social media but I don't need to understand this I guess :).

So long as its got nothing to do with anti-social media like ex-twidder and fakebook. [end grumpy olde guy rant]
